### ISSUE TITLE
Extraction stop 

### ISSUE THREAD

**Issue by kim27y** [tridivb/slowfast_feature_extractor#23]
I can extract video's feature
but It stops after a few extractions always.
Why this occur and How do I do fix it ㅜㅜ

**Comment by xuxin489** [comment]
me too!  i have to delete the processed file name from the csv file every time it stops...
Have you solved it?

**Comment by xuxin489** [comment]
> me too! i have to delete the processed file name from the csv file every time it stops... Have you solved it?

and when it stops , it always shows :  File "/opt/conda/envs/test_slowfast/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt

**Comment by giangdao1402** [comment]
@xuxin489 Have you figure out the solution ?

**Comment by xuxin489** [comment]
> [@xuxin489](https://github.com/xuxin489) Have you figure out the solution ?
No. I just wrote a script that stops and restarts to process the next video after each one is processed.
